### PR TITLE
Socint 347 cases with interactions

### PR DESCRIPTION
**Pull request by PeterBochel**

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/CaseInteractionDetailsDTO.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CaseInteractionDetailsDTO {
```
Comment (rpcatling) [inline src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/CaseInteractionDetailsDTO.java:14]: I thought this was just plain old CaseInteractionDTO ? (without the Details bit)

**Review by rpcatling — CHANGES_REQUESTED** [review]
minor thing as well as the observation about the extra "Details" in the DTO name

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/CaseInteractionDetailsDTO.java (hunk 1 above)
Comment (PeterBochel) [reply src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/CaseInteractionDetailsDTO.java:14]: Renamed.

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/CaseInteractionDetailsDTO.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  public String interactionSource;
```
[the remark this answers is not in the document]
Comment (PeterBochel) [reply src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/CaseInteractionDetailsDTO.java:16]: Ooops.
Fixed. 
All now private.

**Review by rpcatling — APPROVED** [review]

**Pull request merged**